extensions.json

```diff
@@ -405,6 +420,11 @@
       "id": "korekontrol.saltstack",
       "repository": "https://github.com/korekontrol/vscode-saltstack"
     },
+    {
+      "id": "kortina.vscode-markdown-notes",
+      "repository": "https://github.com/kortina/vscode-markdown-notes.git",
+      "version": "0.0.9"
+    },
```

Comment:
Unfortunately, this extension is licensed under GPL-3.0, and it's currently unclear whether GPL-3.0 extensions are compatible with https://open-vsx.org -- see also https://github.com/open-vsx/publish-extensions/issues/49#issuecomment-646469728

So let's remove it from this PR for now. I've filed https://github.com/open-vsx/publish-extensions/issues/72 to request that we add `kortina.vscode-markdown-notes` once GPL-3.0 compatibility is confirmed.

```suggestion
```